src/android/build.gradle

```diff
@@ -5,10 +5,26 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.4.0'
-        classpath 'com.google.firebase:perf-plugin:1.3.5' 
+        classpath 'com.android.tools.build:gradle:3.6.4'
+        classpath 'com.google.gms:google-services:4.3.14'
+        classpath 'com.google.firebase:perf-plugin:1.4.2'
```

Comment:
Do we wneed this since we are using the BoM?

Reply:
I believe we do because of this: https://firebase.google.com/docs/perf-mon/get-started-android#add-perfmon-plugin